src/__tests__/media-query-test.js

```diff
@@ -210,15 +211,27 @@ describe('Media query tests', () => {
       }
     }
 
+    @Radium({matchMedia})
+    class TestComponent extends Component {
```

Comment:
@bobbyrenwick @alexlande See how I had to add an extra wrapper component here? Is this an acceptable tradeoff for `StyleRoot`? Just want to mention it explicitly, since you are both in favor of this approach.